Readme.md

```diff
@@ -30,7 +41,7 @@ Install
 ```bash
 # Replace my-gameserver with how you wish to name your game
 export NODE_ENV=development
-npm install mage --bootstrap --prefix my-gameserver
+npm install mage --create --prefix my-gameserver
```

Comment:
not exactly sure how prefix works but, if it should always be used, why not making disabling it the option.